### DISCUSSION TITLE
Cannot update child in the Question set

### DISCUSSION THREAD

**Discussion by nidhikachhawa24** [Sunbird-coKreat/Community#75] [Bugs]
Hello All,

I've created a **Question set** and which is currently live.

Following are the steps that I've followed to update the Live Question set:

Made changes in the Child section and was able to successfully sent it for review.
Tried to review the Question set from the Reviewer's workspace, not able to see the updated changes made in the child section.
I tried to check the network panel also, there is no API getting failed.

**I need to understand how we can update the child section in Question set?**

**Comment by parthlawate** [comment]
@Krishnaj20 can you help assign someone to look at this ? Its affecting a production SAAS tenant.

**Reply by arun1205** [reply]
@Aparna-karri591 @krgauraw can u pls help here

**Reply by Krishnaj20** [reply]
@nidhikachhawa24 - Pls share more details such as the version of ED you are using? Are you using workspace or crowd sourcing solution of coKreat? What content type are you creating - by question set, do you mean these are QuML based questions? 
Pls note that complete end to end creation (create + review by reviewer using UI) using workspace isn't yet enabled. However, there are alternate ways to review and approve/reject these content.
@mmanimegalai - fya please

**Reply by nidhikachhawa24** [reply]
@Krishnaj20  @mmanimegalai 
- Currently we're using 4.6 version of ED.
- Question set is created by using workspace
- I know Question set is not visible in the reviewer's workspace (we do the review part by using do_id in the url)
- Once the question set is live, we're trying to update it but updating the content is not happpning.

**Reply by Krishnaj20** [reply]
@nidhikachhawa24 - thanks for sharing this info. Can you check in the pipeline what is the issue? Why is the publish not reflecting the update? Or you can share logs as well which can help community to resolve.

**Comment by mmanimegalai** [comment]
Tagging @pallakartheekreddy

**Reply by arun1205** [reply]
@nidhikachhawa24 This is a known defect which is fixed in 4.10 release. Could u confirm if u r using this release?
https://project-sunbird.atlassian.net/browse/IQ-67

**Reply by nidhikachhawa24** [reply]
@arun1205 
We're currently using 4.6 version.

So That means it will not work on 4.6 unless we upgrade it to 4.10.

Thank you

**Reply by nidhikachhawa24** [reply]
@arun1205 
Since we're not able to upgrade at a time and as this is a bug that is affecting a 5K user deployment, can we get a **PATCH** for this?

**Reply by vinukumar-vs** [reply]
@nidhikachhawa24 AS a community, we can only support for the last 2 previous prod releases. Anything beyond that it will be very difficult to support.

Please refer the PR changes of [IQ-67](https://project-sunbird.atlassian.net/browse/IQ-67 ) to apply on your end.
Please discuss with Inquiry BB for the fixe(@sajeshkayyath). The above IQ-67 JIRA ticket has the code changes.

@mmanimegalai @Krishnaj20 @sajeshkayyath @pallakartheekreddy

**Reply by Krishnaj20** [reply]
locking the conversation as the query is answered.
